### ISSUE TITLE
unexpected jump when selecting html tags

### ISSUE THREAD

**Issue by maxwowpow** [scriptin/brackets-indent-softwraps#5]
Hello, I'm not sure whether this "distortion jump" happens because of conflict with another extension(s), but I've quickly tested the most obvios ones, which enhance the Editor.

_Demo_
http://i.imgur.com/2RP28cz.gifv

_Repro_
Sketch a simple html file with an embedded to _body_ tag, e.g. `<p>`. Fill this tag with a multiline text.
Tag should be now indented (child of **body**).
Click on the opening bracket of the tag "<".

_Expected_
No change in the soft-indenting.

_Happens_
Distortion (actually constriction/kerning) and jump of this tag to the beginning of the line.

_Note_
You could try to repro on the clean _brackets_ Installation, I'll try that later when come to this if needed to pinpoint the bug.

_simple html_

```
<!DOCTYPE html>
<html>
<body>
            <div class="panel-body">
                <br/>
                <br/> Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dol
            </div>
</body>
</html>
```

**Comment by scriptin** [comment]
I released a new version - 0.2.1. Please try it to see if it works now.

**Comment by maxwowpow** [comment]
clean! 👍 now it's perfect. Thank you very much for the rapid actions!
